doc/auditing-firmware.md

```diff
@@ -1,4 +1,8 @@
-# Auditing Firmware
+<!--
+Copyright lowRISC Contributors.
+SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
Not needed. Covered in the dep5. Lints will fail if a copyright header is not present where it's required.

Reply:
Love your dep5 lint, I should have known :wink: